localgovdrupal/localgov_forms
# Fix: Address lookup caching

The newly added [local custodian code](https://github.com/localgovdrupal/localgov_forms/blob/b9d8ce4dcef51170f150b7e4d1f62ffe444109e8/src/Element/AddressLookupElement.php#L58) restriction should also be considered when [opting to use cached results](https://github.com/localgovdrupal/localgov_forms/blob/b9d8ce4dcef51170f150b7e4d1f62ffe444109e8/src/Element/AddressLookupElement.php#L302) from previous address lookups.  This is particularly useful when there are multiple address lookup elements in a form but they are using different local custodian codes.

### Test steps
- Add **two** address lookup elements to a Webform.
- When configuring the above address lookup elements, choose **different** Local authorities from the `Local authority` dropdown.
- Open the Webform and do address lookups on both address lookup elements.  Use the same search string such as "sandown".
- Returned address lists should differ due to the difference in local authorities.
